tools/@aws-cdk/cli-args-gen/lib/cli-type-gen.ts

```diff
@@ -129,9 +130,13 @@ function kebabToPascal(str: string): string {
     .join('');
 }
 
-function normalizeDefault(defaultValue: any): string {
-  if (typeof defaultValue === 'boolean' || typeof defaultValue === 'string') {
-    return String(defaultValue);
+function normalizeDefault(defaultValue: any, type: string): string {
+  const validDefaults = ['boolean', 'string', 'number', 'object'];
+  if (validDefaults.includes(typeof defaultValue)) {
```

Comment:
99% sure typeof default here isn't nullsafe since the type of `null` is `'object'` in typescript https://dev.to/_ravo_lution/why-typeof-null-is-object-181

We should make sure this is null/undefined safe.

Reply:
`JSON.stringify(null)` = 'null' which seems reasonable to me. I also don't think there's any way that `defaultValue` actually is null.